### ISSUE TITLE
ImportError: cannot import name 'docevents' from 'botocore.docs.bcdoc'

### ISSUE THREAD

**Issue by ahmei0** [awslabs/aws-control-tower-customizations#34]
botocore 1.18.0 removed AWS CLI specific doc files. Those were already migrated to the CLI in version 1.18.140.

This is failing codebuild stage.

Please add following awscli upgrade command install_stage_dependencies.sh

```
pip install --upgrade pip
```

https://github.com/awslabs/aws-control-tower-customizations/blob/master/source/bin/codebuild_scripts/install_stage_dependencies.sh

**Comment by SamiMamdouh** [comment]
I'm facing same error 
``ImportError: cannot import name 'docevents' from 'botocore.docs.bcdoc' (/root/.pyenv/versions/3.8.3/lib/python3.8/site-packages/botocore/docs/bcdoc/__init__.py)``
Above script should fix that.
Thanks !

**Comment by groverlalit** [comment]
Hello @SamiMamdouh @ahmei0 
We are testing the fix and plan to publish a new version as soon as possible.

**Comment by jdstuart** [comment]
We're experiencing the same issue. Great to know a fix is in the works to solve this.

**Comment by rjenks** [comment]
(I'm not with Amazon/AWS) As a workaround, until the install_stage_dependencies.sh script is fixed in the main repo, you can make a 1 line change to the buildspec to fix this.  The fix is based on the PR that ahmei0 posted above, not my work.  The PR is to fix the install_stage_dependencies.sh script that we can't easily modify in our environment though.  This workaround changes the buildspec which we do have control over while we wait for the real fix.

- Go to the CodePipeline dashboard and select the "Custom-Control-Tower-CodeBuild" project
- Go to the Build Details tab and scroll down to the Buildspec, click Edit
- Find the "pre_build:" "commands:" section.
- Add the following line just before the "bash install_stage_dependencies.sh $STAGE_NAME" line:
`- pip install --upgrade awscli`

I can't afford downtime for our CT customization pipeline, so I needed to find a workaround.

**Comment by frumania** [comment]
Adding 

```
- pip install --upgrade --user awscli
```

just before each "pre_build" block in the buildspec should fix this:

e.g.

```
...
      - apt-get update && apt-get upgrade -y
      - apt-get install zip wget python-pip libyaml-dev -y
      - pip install --upgrade --user awscli
  pre_build:
...
```

**Comment by rcalvachi** [comment]
I modified the CfCT template for the BuildSpecs: 

StackSetCodeBuild
SCPCodeBuild
CustomControlTowerCodeBuild

Add "\n      - pip install --upgrade awscli\n      - aws --version" to lines 517, 641 and 811.

Do it right after "- apt-get update && apt-get upgrade -y". Then Update the CfCT stack.

**Comment by hunttom** [comment]
Can you please close this ticket if v1.2.1 resolves this issue.

**Comment by BR0kEN-** [comment]
This issue has been resolved in https://github.com/aws/aws-cli/releases/tag/1.18.140.

See:
- https://github.com/aws/aws-cli/commit/3e59ab4291f42be57c97ca2bff8c853d5a7ece93
- https://github.com/aws/aws-cli/commit/5822bf405ba0f7c00a12b56c50ced2a5b66eae8c#diff-9a4a2455143ceb35ef6c5128a33e6ead169e7757a85252d7755fd133669a1cdb

**Comment by groverlalit** [comment]
Issue resolved in release v1.2.1
